### ISSUE TITLE
Swagger generated example does not respect proper casing configuration

### ISSUE THREAD

**Issue by k-schneider** [FastEndpoints/FastEndpoints#280]
Everything is configured for camelCase but the example response comes back PascalCase.

![image](https://user-images.githubusercontent.com/21957194/195931804-a6c087c4-4f99-417b-a0d5-a57a6880ecb5.png)

![image](https://user-images.githubusercontent.com/21957194/195931853-f649e825-d7c5-4184-80b7-95abb9e394a2.png)

**Comment by dj-nitehawk** [comment]
just tried with `v5.2` and it's working correctly:

![image](https://user-images.githubusercontent.com/7043768/195968660-097c0659-9852-4bbc-89d1-a90f8ef4bfc4.png)

```cs
var builder = WebApplication.CreateBuilder(args);
builder.Services.AddFastEndpoints();
builder.Services.AddSwaggerDoc();

var app = builder.Build();
app.UseAuthorization();
app.UseFastEndpoints();

app.UseOpenApi();
app.UseSwaggerUi3(c => c.ConfigureDefaults());
app.Run();

public class Endpoint : Endpoint<Request>
{
    public override void Configure()
    {
        Post("test");
        AllowAnonymous();
        Summary(s => s.ExampleRequest = new Request
        {
            Id = "100",
            Balance = 200
        });
    }

    public override Task HandleAsync(Request r, CancellationToken c)
    {
        return SendOkAsync();
    }
}

public class Request
{
    public string Id { get; set; }
    public decimal Balance { get; set; }
}
```

**Comment by k-schneider** [comment]
@dj-nitehawk sorry I should have been more specific.  The issue is with response examples.  I modified your code above to illustrate the issue:

```csharp
var builder = WebApplication.CreateBuilder(args);
builder.Services.AddFastEndpoints();
builder.Services.AddSwaggerDoc();

var app = builder.Build();
app.UseAuthorization();
app.UseFastEndpoints();

app.UseOpenApi();
app.UseSwaggerUi3(c => c.ConfigureDefaults());
app.Run();

public class Endpoint : Endpoint<Request, Response>
{
    public override void Configure()
    {
        Post("test");
        AllowAnonymous();
        Summary(s => {
            s.ExampleRequest = new Request
            {
                Id = "100",
                Balance = 200
            };
            s.ResponseExamples[200] = new Response
            {
                Foo = "hello",
                Bar = "world"
            };
        });
    }

    public override async Task HandleAsync(Request r, CancellationToken c)
    {
        await SendOkAsync(new Response { Foo = "a", Bar = "b" });
    }
}

public class Request
{
    public string Id { get; set; }
    public decimal Balance { get; set; }
}

public class Response
{
    public string Foo { get; set; }
    public string Bar { get; set; }
}
```

![image](https://user-images.githubusercontent.com/21957194/196183026-dc0d798d-ce8a-4fb4-a45c-68f7252ba0ec.png)

**Comment by dj-nitehawk** [comment]
got it...
fixed in `v5.2.1.3-beta`
thanks!
